### PR TITLE
Rm: non serializable exeption - 3x

### PR DESCRIPTION
**Pull request by s2oBCN**
Add NonCPS to solve serializable exception
from https://github.com/opendevstack/ods-jenkins-shared-library/issues/645

### CODE REVIEW

**Comment by clemensutschig** [comment]
@s2oBCN Cool.. does that do  now?

**Comment by s2oBCN** [comment]
> @s2oBCN Cool.. does that do now?

Now it's working fine

File: src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy [hunk 1]
```diff
@@ -1597,9 +1604,7 @@ class LeVADocumentUseCase extends DocGenUseCase {
             def latestValidVersionId = this.getLatestDocVersionId(documentType, [environment])
             def docHistory = new DocumentHistory(this.steps, new Logger(this.steps, false), environment, documentName)
             def docChapters = this.project.getDocumentChaptersForDocument(documentType)
-            def docChapterKeys = docChapters.collect { chapter ->
-                chapter.key
-            }
+            def docChapterKeys = collectDocChaptersKeys(docChapters)
```
Comment (metmajer) [inline src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy:1607]: Same here. `compute`?

**Review by metmajer — CHANGES_REQUESTED** [review]

**Comment by clemensutschig** [comment]
@metmajer - we go now the extra route .. more tests, more components, more everything

**Review by clemensutschig — APPROVED** [review]

File: src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy [hunk 2]
```diff
@@ -894,6 +887,19 @@ class LeVADocumentUseCase extends DocGenUseCase {
         return uri
     }
 
+    @NonCPS
+    private List<Map> testIssuesTestForTCP(List<Project.JiraDataItem> testIssues) {
```
Comment (clemensutschig) [inline src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy:891]: `compute` plesase - for consistency ..

**Pull request merged**